api/templates/index.html

```diff
@@ -3,8 +3,7 @@
 
 {% block main %}
   <div id="app">
-    <div class="centered-container">
-      <h4>You need to enable Javascript in your browser to use this app</h4>
+    <div>
```

Comment:
The inner div doesn't do anything, we can remove it now.
Also, I'm curious as to why you removed the `You need to enable javascript in your browser` line as it helps to know where things go wrong.

Reply:
`You need to enable javascript in your browser` kept showing up before the LoginPage component mounts so I decided to remove it. I also figured it wasn't really of much use as I would know if something goes wrong when a component doesn't mount. Also, somehow my react pages failed to work properly when i removed the inner div so I left it.